docs/resources/resource.md

```diff
@@ -52,9 +52,19 @@ resource "twingate_resource" "resource" {
     }
   }
 
-  access {
-    group_ids = [twingate_group.aws.id]
-    service_account_ids = [twingate_service_account.github_actions_prod.id]
+  dynamic "access_group" {
```

Comment:
Can we add an example of a single `access_group` block as well? While it might not be needed, it would be nice to have an explicit example of a single `access_group` block being used.